terraform/paas/terraform_qa.tfvars

```diff
@@ -0,0 +1,15 @@
+app = {
+  name         = "qa-publish-postgraduate-teacher-training"
+  docker_image = "dfedigital/publish-teacher-training:latest"
+  hostname     = "qa-publish-postgraduate-teacher-training"
```

Comment:
Can you test with `www.qa.publish-postgraduate-teacher-training`? It would be more consistent with the existing environments.

Reply:
You can drop the `postgraduate`

Reply:
Actually it should be: `www.qa.publish-teacher-training-courses`

Reply:
The hostname must be simple. We agreed on `qa-publish-teacher-training-courses`

Reply:
the outdated file is displayed here.